components/sslh/patches/00-makefile.patch

```diff
@@ -0,0 +1,23 @@
+--- Makefile.patched	Tue Mar 10 09:51:39 2015
++++ sslh-v1.17/Makefile	Fri Mar 18 11:34:46 2016
+@@ -5,9 +5,9 @@
+ USELIBWRAP?=	# Use libwrap?
+ USELIBCAP=	# Use libcap?
+ COV_TEST= 	# Perform test coverage?
+-PREFIX=/usr/local
++PREFIX=/usr
+ 
+-MAN=sslh.8.gz	# man page name
++MAN=sslh.8	# man page name
+ 
+ # End of configuration -- the rest should take care of
+ # itself
+@@ -59,7 +59,7 @@
+ 	$(CC) $(CFLAGS) $(LDFLAGS) -o echosrv echosrv.o probe.o common.o $(LIBS)
+ 
+ $(MAN): sslh.pod Makefile
+-	pod2man --section=8 --release=$(VERSION) --center=" " sslh.pod | gzip -9 - > $(MAN)
++	/usr/perl5/5.22/bin/pod2man --section=8 --release=$(VERSION) --center=" " sslh.pod > $(MAN)
```

Comment:
Don't do this, this will be broken when perl is updated. Set PATH in Makefile to /usr/gnu/bin:/usr/bin:/usr/perl5/bin